pwndbg/regs.py

```diff
@@ -280,7 +280,7 @@ def __getattr__(self, attr):
 
             value = int(value)
             return value & pwndbg.arch.ptrmask
-        except (ValueError, gdb.error):
+        except (ValueError, AttributeError, gdb.error) as e:
```

Comment:
Why are we catching `AttributeError` here? When is this actually thrown? This sounds like another issue we're hiding